### PR TITLE
Add esmini to tools page

### PR DESCRIPTION
**Pull request by eknabevcc**

### CODE REVIEW

File: static/assets/tools.json [hunk 1]
```diff
@@ -4854,5 +4854,30 @@
 	"fmuImport": [
               "CS"
 	]
-     }
+    },
+    {
+    "name": "esmini",
+    "license": "osi",
```
Comment (ClemensLinnhoff) [inline static/assets/tools.json:4860]: Shouldn't this be MPL-2,0?

Comment (eknabevcc) [reply static/assets/tools.json:4860]: In the [guide](https://github.com/modelica/fmi-standard.org/blob/main/CONTRIBUTING.md#updating-the-tools-list) I see only two options: "commercial", "osi"

So I guess at this level it's just a distinction between Commercial and Open Source.

Comment (ClemensLinnhoff) [reply static/assets/tools.json:4860]: Oh I was thinking of Open Simulation Interface (OSI) :)
That's why I was a bit confused.

Comment (eknabevcc) [reply static/assets/tools.json:4860]: That was my initial confusion as well, but now I learned a new interpretation of the word OSI :)

**Pull request merged**